pkg/controller/servicebindingrequest/binder_test.go

```diff
@@ -171,6 +171,35 @@ func TestBinderNew(t *testing.T) {
 		require.Equal(t, 0, len(list))
 	})
 
+	t.Run("appendEnvFrom-removeEnvFrom-with-configMapRef", func(t *testing.T) {
+		binder := newBinder(
+			context.TODO(),
+			f.FakeDynClient(),
+			sbr,
+			[]string{},
+			testutils.BuildTestRESTMapper(),
+		)
+
+		require.NotNil(t, binder)
+		secretName := "secret"
+		d := mocks.DeploymentMock("binder", "binder", map[string]string{})
+		envFrom := d.Spec.Template.Spec.Containers[0].EnvFrom
+		envFrom = append(envFrom, corev1.EnvFromSource{
+			ConfigMapRef: &corev1.ConfigMapEnvSource{
+				LocalObjectReference: corev1.LocalObjectReference{
+					Name: "test-configmap",
+				},
+			},
+		})
+
+		list := binder.appendEnvFrom(envFrom, secretName)
+		require.Equal(t, 2, len(list))
+		require.Equal(t, secretName, list[1].SecretRef.Name)
```

Comment:
There are two items in `list` -> `require.Equal(t, 2, len(list))`

We should also have a `require.Equal` check for for `list[0]`